mhctools/cli/args.py

```diff
@@ -128,6 +128,11 @@ def add_mhc_args(arg_parser):
         default="",
         help="Comma or space separated list of allele (default HLA-A*02:01)")
 
+    mhc_options_arg_group.add_argument(
+        "--do-not-raise-on-error",
+        action="store_true", default=False,
+        help="If this arg is present, will not crash on error. Only applies to IEDB predictors")
```

Comment:
maybe indicate what kind of errors this will not crash on, and what the results will be in that case? I.e. is this just going to skip all the peptides that we can't predictor for, or return NaN values, or something else?

Reply:
Hopefully I clarified the comment there - it's the most generic error catch (any Exception), because sometimes it's HTTP errors, sometimes ValueErrors from missing predictions (which sometimes result from HTTP errors), etc. The spirit of this is turning IEDB predictors into a best-effort type of thing, which I'm no longer 100% sure is the right direction but that was the idea. What do you think?